service_configuration_lib/spark_config.py

```diff
@@ -78,7 +77,8 @@
 
 SUPPORTED_CLUSTER_MANAGERS = ['kubernetes', 'local']
 DEFAULT_SPARK_RUN_CONFIG = '/nail/srv/configs/spark.yaml'
-PREFERRED_SPARK_UI_PORT = 39091
+PREFERRED_SPARK_UI_PORT_START = 39091
+PREFERRED_SPARK_UI_PORT_END = 39100
```

Comment:
I'm thinking to move these in to srv-confgs, so the jupyter server api handler can also read them

Reply:
srv-configs #27930